src/components/space/ActionsMap.js

```diff
@@ -55,7 +55,11 @@ function ActionsMap() {
   // e.g. we retrieve 100 total actions and 10 users, but these 10 users have only made 90 actions
   // therefore, to avoid confusion: when all users are selected, show all actions
   let actionsToChart;
-  if (usersToFilter.length === 0 || usersToFilter.length === allUsers.length) {
+  if (
+    usersToFilter === null ||
```

Comment:
`!usersFilter` should take care of the first two cases.

Reply:
If you have `usersToFilter = []`, `!usersToFilter` evaluates to `false`, i.e. `usersToFilter` is truthy even though it's an empty array. So with arrays you need to explicitly check for `if (myArray.length)`/`if (!myArray.length)`.

On the other hand, if `usersToFilter = null`, both `usersToFilter.length` and `!usersToFilter.length` will give `TypeError` (so you can't use the shortcut `if (!usersToFilter.length)`).

(I updated the comments in these components to explain why I added the `null` conditional)